state-sync/src/executor_proxy.rs

```diff
@@ -13,6 +13,7 @@ use diem_types::{
     contract_event::ContractEvent,
     ledger_info::LedgerInfoWithSignatures,
     move_resource::MoveStorage,
+    on_chain_config,
```

Comment:
remove?

Reply:
It's used for `on_chain_config::new_epoch_event_key()`